src/featurestorebundle/delta/TableCreator.py

```diff
@@ -30,4 +36,13 @@ def create_if_not_exists(self, full_table_name: str, path: str, schema: StructTy
 
         df.write.partitionBy(*partition_by).format("delta").option("path", path).saveAsTable(f"{full_table_name}")
 
+        self.__set_delta_name_column_mapping_mode(table_identifier=full_table_name)
+
         self.__logger.info(f"Table {full_table_name} successfully created")
+
+    def __set_delta_name_column_mapping_mode(self, table_identifier: str):
+        self.__table_properties_setter.set_properties(
+            table_identifier=table_identifier,
+            property_names=["delta.minReaderVersion", "delta.minWriterVersion", "delta.columnMapping.mode"],
+            property_values=["2", "5", "name"],
```

Comment:
Table properties are key value pairs right? Wouldn't be better to pass a dict instead of two arrays?

Reply:
Yes, definitely :+1: